pom.xml

```diff
@@ -77,6 +77,12 @@
             </exclusions>
         </dependency>
 
+        <!-- Spring web -->
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-web</artifactId>
```

Comment:
Wonder do we need to consider using some other http components instead of `spring-web`, e.g., [apache httpcomponents](https://hc.apache.org/httpcomponents-client-ga/).

Reply:
I prefer to spring-web instead of apache, as if we leverage apache, we need to choose one version which may pollute user space dependency. But for spring-web, we can follow the same version with springframework.